src/App/App.js

```diff
@@ -1,23 +1,24 @@
-import React, { useState, useEffect } from 'react'
+import React, { useState, useEffect, useContext } from 'react'
```

Comment:
The `React` global was [removed from React](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html) in 2020, so I'd avoid using it in new code.

Reply:
Thanks! 👍 Redundant `import React`s have been removed.